build.gradle

```diff
@@ -4,6 +4,9 @@ plugins {
     id 'com.google.protobuf' version '0.9.4';
 }
 
+if (!gradle.gradleVersion.startsWith('8')) // TODO: this breaks as soon as gradle 9 is released, also, I think gradle 7 and 6 works as well.
```

Comment:
Try gradle 6 and 7 (use a docker container) and simply add more `startsWith` clauses. It might be ugly, but it’ll work for now